test/api_tests/var.lua

```diff
@@ -387,5 +387,73 @@ test:is_deeply(compiled.get_types(),
         "union_type","union_value", "array*","map","fixed*"},
     "compiled.get_types")
 
+
+-- Schema evolution: extend a schema with a record field of type
+-- union or record with a default value.
```

Comment:
1. Split comment from a code block below to show it is related to several code blocks below.
2. Are default union case tested for one schema? The code under hood is different for one and two schemas cases, so maybe it worth to do.

Reply:
1. Ok

Reply:
2. I have only changed the code which works in case the second schema has an extra field. (Case for a single schema is not affected)

Reply:
2. Alex opened the issue: https://github.com/tarantool/avro-schema/issues/102